packages/@angular/cli/blueprints/ng/files/package.json

```diff
@@ -12,11 +12,13 @@
   },
   "private": true,
   "dependencies": {
+    "@angular/animations": "^4.0.0",
```

Comment:
I understand that, which is why a flag makes sense for it. Where I am confused is what benefit does it provide to include on the `package.json` which would download it to offline caches and such for projects that may not use animations at all.

Also if its preferred to always have `@angular/animations` whether is needed or not for https://github.com/angular/angular-cli/issues/5928 then we could at least re-open https://github.com/angular/angular-cli/issues/5785 to tackle optionally importing browser animations.